services/app-web/src/pages/StateSubmission/RateDetails/RateDetails.tsx

```diff
@@ -197,7 +199,8 @@ export const RateDetails = ({
                         })
                     }
                 })
-            setPackageOptions(packages)
+            const abbreviatedPackagesList = packages.slice(0, 5)
```

Comment:
I'm seeing the earliest 5 submissions instead of the most recent 5. I wonder if we need to reverse then slice or should we do a sort at an earlier part of this and be confident we have the correct order?

Reply:
Did a few things to address this
- Added the sorting by `updatedAt` quick fix [cf17705](https://github.com/CMSgov/managed-care-review/pull/1349/commits/cf1770534403f207c93ce65bbea6c6b06b008952). 
- Looked more into react-select when I realized the only way to limit the visible options _but still show all the options in the typeahead_ is not by limiting the options list like I was doing but adding more dynamic handling. As a result looked into AsyncSelect with defaultOptions https://react-select.com/async#defaultoptions but decided it was a lot of code (we end up having to re-implement the search logic rather than using built in behavior) for what is basically a styling motivated change.
-  Decided to go with shrinking the size of the dropdown instead and also ran that by design [37b1e50](https://github.com/CMSgov/managed-care-review/pull/1349/commits/37b1e5006ddba194ef0af4487ca8ad3dde1b3298) 
- I made sure that how we sort and pick the current revision is always the same [e0e422f](https://github.com/CMSgov/managed-care-review/pull/1349/commits/e0e422fedb4a57973ce112283d4307fd57225f00) Part of what made debugging hard is we have a few ways we were doing this.